engine/execution/ingestion/engine.go

```diff
@@ -178,14 +172,7 @@ func (e *Engine) Process(originID flow.Identifier, event interface{}) error {
 }
 
 func (e *Engine) process(originID flow.Identifier, event interface{}) error {
-	switch resource := event.(type) {
-	case *messages.ExecutionStateSyncRequest:
-		return e.handleStateSyncRequest(originID, resource)
-	case *messages.ExecutionStateDelta:
-		return e.handleStateDeltaResponse(originID, resource)
-	default:
-		return fmt.Errorf("invalid event type (%T)", event)
-	}
+	return nil
```

Comment:
we might keep the invalid event type error here to see if partner nodes are calling this